Reference/Debugging/index.md

```diff
@@ -0,0 +1,71 @@
+---
+versionFrom: 8.1.0
+---
+
+# Debugging
```

Comment:
wondering whether we should call the article 'Debugging with SourceLink' ?

we have a page called 'debugging' already, in getting started - code - debugging...

https://our.umbraco.com/documentation/Getting-Started/Code/Debugging/

and we could add at the footer of that page a link back to this reference page... ?

Reply:
```suggestion
# Debugging with SourceLink
```